airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/payment_source.json

```diff
@@ -152,7 +152,7 @@
         },
         "funding_type": {
           "type": ["string", "null"],
-          "enum": ["credit", "dedit", "prepaid", "not_known", "not_applicable"]
+          "enum": ["credit", "debit", "prepaid", "not_known", "not_applicable"]
```

Comment:
enum can also be removed

Reply:
@artem1205 Would you recommend I remove enumeration validation from all schemas for consistency? Looks like it was originally included when the connector was developed. I imagine this would only cause validation issues on our end.

Reply:
Removed all unnecessary validation properties in the latest commit.